examples/protonect/src/registration.cpp

```diff
@@ -85,6 +90,16 @@ void Registration::apply( int dx, int dy, float dz, float& cx, float &cy) const
   cx = rx * color.fx + color.cx;
 }
 
+/**
+ * Undistort/register a whole image.
+ * @param rgb RGB color image (1920x1080).
+ * @param depth Depth image (512x424)
+ * @param [out] undistorted Undistorted depth image.
+ * @param [out] registered Image Color image fixed for depth.
+ * @param enable_filter Whether or not to drop 'too noisy' pixels.
```

Comment:
Perhaps change this to `@param enable_filter use a depth buffer to remove pixels which are not visible to both cameras`